### PR TITLE
Added stairs and slabs blockstate datagen, and changed blockstate datagen for simple blocks.

### PR DESCRIPTION
**Pull request by GideonHead**
Also added cheese blocks, only accessible in creative (redid this PR because I accidentally made the last one on the 1.19 branch.)

### CODE REVIEW

**Comment by GideonHead** [comment]
Dont merge this yet, tonight I will be adding a lot more datagen for different types of blocks.

**Comment by GideonHead** [comment]
alright, this PR is now finished.

**Review by Dweblenod — COMMENTED** [review]
From a brief inspection of the code on Github, this PR appears solid!

However I am going to request you remove the cheese blocks from this PR. We will need to discuss the plan with them. It should be feasible to make a copy of this branch, then revert cheese related content in commits on this branch while leaving the other one for a different day.

File: src/main/java/com/mraof/minestuck/block/MSBlocks.java [hunk 1]
```diff
@@ -486,7 +486,6 @@ private static Block coalOre(BlockBehaviour.Properties properties)
 	public static final RegistryObject<Block> DEAD_PLANKS_SLAB = REGISTER.register("dead_planks_slab", () -> new SlabBlock(copy(DEAD_PLANKS.get())));
 	public static final RegistryObject<Block> TREATED_PLANKS_SLAB = REGISTER.register("treated_planks_slab", () -> new SlabBlock(copy(TREATED_PLANKS.get())));
 	
-
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/block/MSBlocks.java:489]: Small nitpick, please reverse this change!

**Comment by GideonHead** [comment]
note for dweb: not sure if you saw cuz you were gone yesterday, but I answered your questions in the cheese lands thread.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 - Grist cost added for stripped bamboo block group, suspicious blocks, sniffer egg, pink petals, torchflower, torchflower seed, pitcher plant, pitcher pod, relic music disc
 - Uncarved wood block group now has a stonecutting recipe and grist cost
+- American Cheese, Smooth American Cheese, Swiss Cheese and Smooth Swiss Cheese, along with Stairs and Slabs for each
```
Comment (Dweblenod) [inline CHANGELOG.md:13]: Remove this line

Also remove Vinnelli as contributor if their contribution was the no longer included art

File: src/main/java/com/mraof/minestuck/item/MSCreativeTabs.java [hunk 3]
```diff
@@ -474,6 +474,9 @@ private static void buildLandsTab(CreativeModeTab.ItemDisplayParameters paramete
 		output.accept(MSItems.CARVED_PLANKS.get());
 		output.accept(MSItems.POLISHED_UNCARVED_WOOD.get());
 		output.accept(MSItems.CARVED_KNOTTED_WOOD.get());
+		output.accept(MSItems.CARVED_KNOTTED_WOOD.get());
+		output.accept(MSItems.CARVED_KNOTTED_WOOD.get());
+		output.accept(MSItems.CARVED_KNOTTED_WOOD.get());
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/item/MSCreativeTabs.java:479]: Remove this!

**Review by Dweblenod — COMMENTED** [review]
Make sure to remove the texture files!

**Comment by GideonHead** [comment]
I have solved all of the issues brought up

**Pull request merged**